pithos/plugins/mediakeys.py

```diff
@@ -68,10 +73,10 @@ def mediakey_signal(self, proxy, sender, signal, param, userdata=None):
                 self.window.playpause_notify()
             elif action == 'Next':
                 self.window.next_song()
-            elif action == 'Stop' or 'Pause':
-                self.window.user_pause()
             elif action == 'Previous':
                 self.window.bring_to_top()
+            elif action in ('Stop', 'Pause'):
```

Comment:
Well now you got your commits messed up, this needs to be rebased into the first one.

Reply:
So just squash it before you push it. Both commit titles will be in the squashed commit message.

Reply:
There is a purpose to preserving history of unrelated bug-fixes.

Reply:
OK I'll work my git magik,lol!!!

Reply:
```sh
git reset HEAD~2
git add -p
# select just the one hunk
git commit -m 'mediakeys: Listen for 'Pause' mediakeys action'
git commit -am 'mediakeys: Workaround MATE not updating it's window state properly'
```